recipe/build_root-base.sh

```diff
@@ -101,6 +101,8 @@ cmake -LAH \
     -Dtesting=ON \
     -Droottest=OFF \
     -Dccache=OFF \
+    -Dbuiltin_ftgl=ON \
```

Comment:
These two packages should be added to conda-forge instead of using the builtins.

Reply:
I can try that, I'm working on testing root7 + BLAS now. Is it always OpenBLAS on conda-forge?

Reply:
Great, thanks!

No, it uses a reference implementation by default so any BLAS can be used, see: https://conda-forge.org/docs/maintainer/knowledge_base.html#blas

Reply:
I can add these next week if you need me to.

Reply:
Then how do I get CMake to find the Conda-Forge BLAS? It seems to be a fairly standard usage of FindBLAS.cmake, from CMake.

Reply:
I take it ROOT uses the library on source-forge, last updated in 2013?

Reply:
There seems to be a more modern fork here: https://github.com/ulrichard/ftgl

Reply:
Looks like ROOT forked it a very long time ago: https://github.com/root-project/root/commits/master/graf3d/ftgl

Homebrew is packaging from sourceforge: https://formulae.brew.sh/formula/ftgl

Reply:
@henryiii, did you get around to adding `ftgl` as well?

Reply:
I have not yet; it would pretty much need to be copied from the ROOT source.

Reply:
There does not even seem to be an intact build system for it in ROOT.

Reply:
Can't the homebrew recipe be used as a reference? Or is it patched in ROOT? https://formulae.brew.sh/formula/ftgl

Reply:
Just some warning fixes and such. I don't see anything too bad so far. https://github.com/root-project/root/commit/0d22cbca72b70e852f8abf876cb998a88dc32813#diff-4c3e24d7189cd86a85ef79d1e6a143dd

Reply:
I'll give it a shot